lib/routes/kuaidi100/utils.js

```diff
@@ -25,12 +33,24 @@ async function getCookie(ctx) {
                     wwwid = e.split(';')[0];
                 } else if (e.indexOf('csrftoken') === 0) {
                     csrf = e.split(';')[0];
+                } else if (e.indexOf('globacsrftoken') === 0) {
+                    globacsrftoken = e.split(';')[0];
+                } else if (e.indexOf('dasddocTitle') >= 0) {
+                    dasddocTitl = e.split(';')[0];
+                } else if (e.indexOf('dasddocReferrer') >= 0) {
+                    dasddocReferrer = e.split(';')[0];
+                } else if (e.indexOf('dasddocHref') >= 0) {
+                    dasddocHref = e.split(';')[0];
                 }
             }
         }
 
         ctx.cache.set(wwwid_key, wwwid, 600);
         ctx.cache.set(csrf_key, csrf, 600);
+        ctx.cache.set(globacsrftoken_key, globacsrftoken, 5);
+        ctx.cache.set(dasddocTitl_key, dasddocTitl, 5);
+        ctx.cache.set(dasddocReferrer_key, dasddocReferrer, 5);
+        ctx.cache.set(dasddocHref_key, dasddocHref, 5);
```

Comment:
Do you mean 5 mins? The unit here is second.

Reply:
Thank you for your comment. I'm not sure. 600 may be better for Cookies.

```suggestion
        ctx.cache.set(dasddocHref_key, dasddocHref, 600);
```

Reply:
But I haven't tested this situation.